### PR TITLE
fix(test): use the new GC schedule for e2e test

### PR DESCRIPTION
**Pull request by youngjoon-lee**
## 1. What does this PR implement?

We have the `sdp_blend_activity` e2e test that checks that the declarations are successfully refreshed by activity messages submitted by the blend service.

The test was waiting `inactivity + retention + 1` period to check if the declarations still survive. But this waiting period must be changed because the [RFC](https://app.notion.com/p/nomos-tech/RFC-Remove-Concept-of-a-Session-f39261aa09df826db83e81613bb454dc?source=copy_link#366261aa09df80f683bfd05ce34a045b) initializes `Declaration.active` with `created + 2`. This PR changes the waiting period to `declaration.active + inactivity + retention + 1`.

## 2. Does the code have enough context to be clearly understood?

yes

## 3. Who are the specification authors and who is accountable for this PR?

@youngjoon-lee 
spec: @madxor 

## 4. Is the specification accurate and complete?

yes

## 5. Does the implementation introduce changes in the specification?

no

## Checklist

> [!WARNING]  
> Do not merge the PR if any of the following is missing:

* [x] 1. The PR title follows the Conventional Commits [specification](https://www.notion.so/How-to-open-a-Pull-Request-215261aa09df80deb538ed59f5e4e0b5).
* [x] 2. Description added.
* [x] 3. Context and links to Specification document(s) added.
* [x] 4. Main contact(s) (developers and specification authors) added
* [x] 5. Implementation and Specification are 100% in sync including changes. This is critical.
* [x] 6. Link PR to a specific milestone.
* [x] 7. New or changed logs were reviewed against the [logging guidelines](https://github.com/logos-blockchain/logos-blockchain/blob/master/CONTRIBUTING.md#logging-guidelines).

### CODE REVIEW

File: tests/src/tests/mantle/sdp/activity.rs [hunk 1]
```diff
@@ -105,7 +102,8 @@ fn test_config(mut config: RunConfig, slots_per_epoch: &AtomicU64) -> RunConfig
     };
     config.deployment.cryptarchia.security_param = NonZero::new(2).unwrap();
     config.deployment.cryptarchia.slot_activation_coeff =
-        NonNegativeRatio::new(1, 10.try_into().unwrap());
+        NonNegativeRatio::new(1, 2.try_into().unwrap());
+    config.deployment.cryptarchia.learning_rate = 0.5.try_into().unwrap();
```
Comment (ntn-x2) [inline tests/src/tests/mantle/sdp/activity.rs]: Why this change? Or these changes in general?

**Review by ntn-x2 — APPROVED** [review]

File: tests/src/tests/mantle/sdp/activity.rs (hunk 1 above)
Comment (youngjoon-lee) [reply tests/src/tests/mantle/sdp/activity.rs]: I increased them to make the test faster, but now I think we don't need to increase the learning rate. I will revert it.
